pkg/services/sqlstore/migrations/short_url_mig.go

```diff
@@ -0,0 +1,18 @@
+package migrations
+
+import . "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+
+func addShortUrlMigrations(mg *Migrator) {
+	shortUrlV1 := Table{
+		Name: "short_url",
+		Columns: []*Column{
+			{Name: "uid", Type: DB_NVarchar, Length: 40, Nullable: false, IsPrimaryKey: true},
```

Comment:
Would suggest adding `id` and `org_id`. `id` to be auto increment int64 and primary key for more performance optimized primary index that is naturally sorted. `org_id` to make sure always filter by org_id since Grafana is multi tenant. You probably want to add a unique index of org_id and uid to improve the performance for lookup a short url.